### PR TITLE
post-merge updates: intro to ether page

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

updates `developers/docs/intro-to-ether/index.md` ready for post-merge

## Related Issue

#7075

### CODE REVIEW

File: src/content/developers/docs/intro-to-ether/index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insi
 
 Minting is the process in which new ether gets created on the Ethereum ledger. The underlying Ethereum protocol creates the new ether, and it is not possible for a user to create ether.
 
-Ether is minted when a new block is created on the Ethereum blockchain. As an incentive to build blocks, the protocol grants a reward in each block, incrementing the balance of an address set by the block producer. The block reward has changed over time, and today it is 2 ETH per block. After the merge issuance to each validator depends upon the amount of ether they have staked and their performance.
+Ether is minted in each slot (every 12 seconds). The total amount issued depends on the number of validators and how much ether they have staked. This total issuance is divided equally among validators in the ideal case that all validators are honest and online, but in reality it varies based on validator performance. About 1/8 of the total issuance goes to the block proposer, the remainder is distributed across the other validators. Block proposers also receive tips from transaction fees and MEV related income but these come from recycled ether, not fresh issuance.
```
Comment (wackerow) [inline src/content/developers/docs/intro-to-ether/index.md:37]: > Ether is minted in each slot (every 12 seconds). 

Would push back on this a bit. Correct me if I'm wrong, but ether isn't necessarily minted each "slot"... Each slot is an opportunity for a block, but is not guaranteed (ie, block proposer is offline), and IIRC, attestation rewards are not credited to the validator balance until the epoch checkpoint.

Perhaps:
```suggestion
Ether is minted as a reward for each block proposed, and also at every epoch checkpoint for other validator activity related to reaching consensus. The total amount issued depends on the number of validators and how much ether they have staked. This total issuance is divided equally among validators in the ideal case that all validators are honest and online, but in reality it varies based on validator performance. About 1/8 of the total issuance goes to the block proposer, the remainder is distributed across the other validators. Block proposers also receive tips from transaction fees and MEV related income but these come from recycled ether, not fresh issuance.
```

Not sure if we should get into block/slot timing in this section, any thoughts?

**Review by wackerow — CHANGES_REQUESTED** [review]
@jmcook1186 Left a comment...

Also, I'm assuming it wasn't intentional to include changes to the `data/translation-reports/**/*.json` files—any chance we can revert these?

**Comment by jmcook1186** [comment]
incorporated your suggestion and reverted changes to `/intl` in forced push.

File: src/content/developers/docs/intro-to-ether/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insi
...
 
-Ether is minted when a new block is created on the Ethereum blockchain. As an incentive to build blocks, the protocol grants a reward in each block, incrementing the balance of an address set by the block producer. The block reward has changed over time, and today it is 2 ETH per block. After the merge issuance to each validator depends upon the amount of ether they have staked and their performance.
+Ether is minted as a reward for each block proposed, and also at every epoch checkpoint for other validator activity related to reaching consensus. The total amount issued depends on the number of validators and how much ether they have staked. This total issuance is divided equally among validators in the ideal case that all validators are honest and online, but in reality it varies based on validator performance. About 1/8 of the total issuance goes to the block proposer, the remainder is distributed across the other validators. Block proposers also receive tips from transaction fees and MEV related income but these come from recycled ether, not fresh issuance.
```
Comment (minimalsm) [inline src/content/developers/docs/intro-to-ether/index.md:37]:
```suggestion
Ether is minted as a reward for each block proposed and at every epoch checkpoint for other validator activity related to reaching consensus. The total amount issued depends on the number of validators and how much ether they have staked. This total issuance is divided equally among validators in the ideal case that all validators are honest and online, but in reality, it varies based on validator performance. About 1/8 of the total issuance goes to the block proposer; the remainder is distributed across the other validators. Block proposers also receive tips from transaction fees and MEV-related income, but these come from recycled ether, not new issuance.
```

**Review by minimalsm — APPROVED** [review]

**Pull request merged**